src/soft-tabs-transform.coffee

```diff
@@ -0,0 +1,26 @@
+Point = require './point'
+Region = require './region'
+_ = require 'underscore-plus'
+
+module.exports =
+class SoftTabsTransform
+  constructor: (@tabLength) ->
+    @tab = _.multiplyString(" ", @tabLength)
```

Comment:
I would avoid suffixing it with `String`: usually I find it somewhat redundant to include the type in a variable name.

What do you think if we just call it `softTab`?